Code/GraphMol/QueryAtom.h

```diff
@@ -64,6 +64,19 @@ class RDKIT_GRAPHMOL_EXPORT QueryAtom : public Atom {
     }
     return *this;
   }
+
+  QueryAtom(QueryAtom &&other) noexcept : Atom(std::move(other)) {
+    dp_query = std::move(other.dp_query);
```

Comment:
Stylistically should this be in the initializer args?

```
QueryAtom(QueryAtom &&other) noexcept : 
  Atom(std::move(other)), 
  dp_query(std::move(other.dp_query) {
}
```

Reply:
actually I think we should be using std::exchange here too